hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/RatisUtil.java

```diff
@@ -76,7 +76,8 @@ public static void setRaftStorageDir(final RaftProperties properties,
                                        final ConfigurationSource conf) {
     String storageDir = haConf.getRatisStorageDir();
     if (Strings.isNullOrEmpty(storageDir)) {
-      storageDir = ServerUtils.getDefaultRatisDirectory(conf);
+      File metaDirPath = ServerUtils.getOzoneMetaDirPath(conf);
+      storageDir = (new File(metaDirPath, "scm-ha")).getPath();
```

Comment:
I think it's prefered to use SCMHA Configuration to set the storageDir

Reply:
agree

Reply:
After https://github.com/apache/ozone/pull/1739 this can be set through HA config.